style.css

```diff
@@ -417,5 +451,149 @@ hr {
   width: 140px;
   float: left;
   margin-top: 1px;
-  margin-right: 220px;
+  margin-right: 200px;
+}
+
+@media (min-width: 748px) {
+  section {
+    background-position: 20px 0;
+  }
+
+  .header-section {
+    width: 100%;
+  }
+
+  .main-part {
+    min-width: 1100px;
+    padding-top: 160px;
+    padding-bottom: 260px;
+  }
+
+  .my-logo {
+    margin-left: 80px;
+  }
+
+  .navigation-links {
+    display: inline;
+    margin-right: 40px;
+  }
+
+  .navigation-right {
+    display: none;
+  }
+
+  .welcome-message {
+    margin-left: 160px;
+  }
+
+  .about-me-main {
+    font-size: 16px;
```

Comment:
- [ ] Consider using the correct font size like the one in the Figma for the paragraph, it will be `20px`.